src/lib/pages/frontsite/Documents/DocumentDetails/DocumentDetails.js

```diff
@@ -160,7 +160,7 @@ class DocumentDetails extends Component {
         params: { documentPid },
       },
     } = this.props;
-    if (hasError && error.response.status === 404) {
+    if (hasError && error.response && error.response.status === 404) {
```

Comment:
```suggestion
    if (hasError && _has(error.response.status) === 404) {
```
do not forget to import _has from lodash